### ISSUE TITLE
Reclaim process - next steps?

### ISSUE THREAD

**Issue by pattM** [iotaledger/wallet#531]
Hi, I would like to ask if someone knows more about the second phase of reclaim process. My old address was in the snapshot with the correct balance and was marked as CURL_UNUSED (I missed the transition in the past). I did the first phase of reclaim in the wallet 2.5.3 with success, then I continued with the second phase with wallet 2.5.4 also with success. Now it's more than 2 weeks since that and I still don't have my iota on my new address. I read the blog about that iota will be sending to holders starting from November 13th on first come, first serve manner. I can see 2 incoming transactions on my old address and balance zero (which is I think correct as snapshot happened), but on my new address I can see only one incoming transaction (which is the proof transaction) and balance is still zero. So I would like to ask whether I should wait for the next wallet or what will be the next steps? Thank anyone for some information.

**Comment by abihsot** [comment]
did you clicked multiple times on attach to tangle (in receive section) till you see your old balance, if not, do it now. Worked for me.

**Comment by pattM** [comment]
Yes, I tried it multiple times, but still with no success. The address where the IOTA foundation should send my "saved" IOTA after reclaim process is already attached to tangle, but with 0 balance. How many times did you try re-attach to tangle?

**Comment by wonkytonky** [comment]
apparently it's a manual process  for foundation to send money to  claimed address.. . so it can take time

**Comment by pattM** [comment]
Thank you for your answer (sorry it's a bit late)...But can someone confirm your words? Because I haven't seen any blogspot or anything about this issue and as I see, I'm not the only one :(
